DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/stats/stats.factory.js

```diff
@@ -0,0 +1,40 @@
+﻿(function () {
+    'use strict';
+    var module = angular.module('app.stats');
+
+    module.factory('statsFactory', statsFactory);
+
+    statsFactory.$inject = ['$http'];
+
+    function statsFactory($http) {
+        var baseUrl = 'api/user';
```

Comment:
if we don't have the place for constants - we need service of type constant with all constants!
And also as I remember we have base url as follows: `api/v1/`.
Imagine: we will change base url to `api/v2/`. So we will need to rewrite everything!